packages/dd-trace/src/appsec/telemetry.js

```diff
@@ -69,13 +77,20 @@ function getOrCreateMetricTags (req, versionsTags) {
 }
 
 function updateWafRequestsMetricTags (metrics, req) {
-  if (!req || !enabled) return
+  if (!req) return
+
+  const store = getStore(req)
+
+  // it does not depend on whether telemetry is enabled or not
```

Comment:
The comment is ok, but I would appreciate a test to check that it works even with metrics disabled to prevent futures changes here